README.md

```diff
@@ -152,18 +152,52 @@ These additional options are specific to pug templates and can be configured in
 
   Choices:
 
-  - `'keep-all'` _default_ -> Keep all spaces within comments.  
+  - `'keep-all'` _default_ → Keep all spaces within comments.  
     Example: `// ___this _is __a __comment_`
-  - `'keep-leading'` -> Keep leading spaces within comments.  
+  - `'keep-leading'` → Keep leading spaces within comments.  
     Example: `// ___this is a comment`
-  - `'trim-all'` -> Trim all spaces within comments.  
+  - `'trim-all'` → Trim all spaces within comments.  
     Example: `// this is a comment`
 
 - `pugSortAttributesBeginning` & `pugSortAttributesEnd`  
   Sort attributes by regex patterns to the beginning or the end.  
   [Example](https://github.com/prettier/plugin-pug/issues/22#issuecomment-699509995)  
   _This feature was planned since `1.2.0`, but it was always a bit unstable and opinionated._  
   _If there are any bugs, please report them._
+- `pugWrapAttributesThreshold`  
+  Define the maximum amount of attributes that an element can appear with on one line before it gets wrapped.
+
+  Choices:
+
+  - `-1` _default_ → Only wrap attributes as needed.  
+    Example:
+    ```pug
+    input(type="text")
+    input(type="text", value="my_value", name="my_name")
+    ```
+  - `0` → Always wrap attributes.  
+    Example:
+    ```pug
+    input(
+      type="text"
+    )
+    input(
+      type="text",
+      value="my_value",
+      name="my_name"
+    )
+    ```
+  - `1` → Allow one unwrapped attribute, wrap two and more.  
+    Example:
+    ```pug
+    input(type="text")
+    input(
+      type="text",
+      value="my_value",
+      name="my_name"
+    )
+    ```
+  - `2 … Infinity` → Same as above, just with different thresholds.  
 
 The definitions for these options can be found in [src/options/index.ts](https://github.com/prettier/plugin-pug/blob/master/src/options/index.ts)
```

Comment:
Could you also change this? `src/options/index.ts` is outdated :thinking: 
Maybe `src/options/*`? Or leave it completely?!

Reply:
ok, leave it out

Reply:
Done!